functions/stats.ts

```
@@ -0,0 +1,298 @@
+    d1.getMonth() === d2.getMonth() &&
+    d1.getDate() === d2.getDate();
+}
+
+/**
+ * Get player's highest number of tags in one day data
+ * @param player Player to get highest number of tags in one day data
+ * @returns Highest number of tags per day data
+ */
+function getPlayerHighestNumberTagsPerDayData(
+  player: Player
+): {} {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)